sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorLogExporter.cs

```diff
@@ -63,5 +64,16 @@ public override ExportResult Export(in Batch<LogRecord> batch)
 
             return exportResult;
         }
+
+        protected override void Dispose(bool disposing)
+        {
+            if (!_disposed)
```

Comment:
Should we check for `disposing` is true? Like this one - https://github.com/open-telemetry/opentelemetry-dotnet/blob/690f7e5bbcf2cbcaba47b595b16446ac628377b9/src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs#L241

Reply:
Yes, updated